### DISCUSSION TITLE
providing a Gitpod configuration

### DISCUSSION THREAD

**Discussion by NicoWeio** [ManimCommunity/manim#2127] [Suggestions and Proposals]
Today, I spent a few hours trying to automate the setup of a dev environment for Manim using [gitpod.io](https://gitpod.io), taking inspiration from https://github.com/ManimCommunity/manim/blob/main/.github/workflows/ci.yml and https://github.com/ManimCommunity/manim/blob/main/docker/Dockerfile.
While I was able to build/execute Manim in the generated container after some tries, I can't get it quite right. 

Do you agree that providing a Gitpod (or GitHub Codespaces in the future) configuration is a good idea? In this case, can someone with more experience in Manim/Gitpod/Docker build a proper one? :)

**Comment by Darylgolden** [comment]
Yes, it's a good idea!
